pantheon-bundle/src/main/java/com/redhat/pantheon/servlet/module/PublishDraftVersion.java

```diff
@@ -75,64 +84,74 @@ public void run(SlingHttpServletRequest request, PostResponse response, SlingPos
             asciidoctorService.getModuleHtml(module, locale, variant, false, new HashMap(), true);
             events.fireEvent(new ModuleVersionPublishedEvent(moduleVersion), 15);
         }
+        log.debug("Operation Publishinging draft version,  completed");
+        long elapseTime = System.currentTimeMillis() - startTime;
+        log.info("Total elapsed http request/response time in milliseconds: " + elapseTime);
```

Comment:
Do we need this as info rather than debug? Just curious.

Reply:
Changed level of logger to debug,